deploy/cr.yaml

```diff
@@ -56,3 +56,20 @@ spec:
     #  destination: file
     #  format: BSON
     #  filter: '{}'
+  backup:
+    enabled: false
+    version: 0.1.0
+    restartPolicy: OnFailure
```

Comment:
This would allow the user to specify if they want to backup to restart on failure or not. Perhaps a boolean flag like 'restartOnFailure: true' would be better @delgod?

Reply:
yea, I think boolean flag will be better. thank you!